client/sections.js

```diff
@@ -464,6 +474,30 @@ const sections = [
 		secondary: true,
 		group: 'sites',
 	},
+	{
```

Comment:
I don't see why these sections should be here, given that Calypso won't be using them. We could take advantage of code splitting and still define those only for Jetpack Cloud.

Reply:
If the sections are not defined here no section files get created. I think it probably a good idea to create a single way to get sections for each of the different usecases.